tests/lib/nsxv_test.py

```diff
@@ -234,6 +234,40 @@
   }
   """
 
+SOURCE_IPv6_MISMATCH = """\
```

Comment:
I think naming these a bit longer is a good idea.

I've been doing:
mixed-to-v6
mixed-to-v4
mixed-to-mixed
v4-to-mixed
v4-to-v4
v4-to-v6
v6-to-v6

Something like that that clarifies the intent of what it is doing.